### ISSUE TITLE
Integration for „Umweltsensor“

### ISSUE THREAD

**Issue by Kohle81** [bombadiltom/homebridge-rademacher-homepilot#5]
Hello,

thanks for your great plugin.

Is it possible to integrate this sensor to homekit with this plugin?

https://www.tecedo.de/rademacher-9475-duofern-umweltsensor-29468.html

**Comment by bombadiltom** [comment]
If you buy one for me, I could integrate it ;-)

**Comment by Kohle81** [comment]
Hi,

it‘ s a nice idea ;-) for me alone it is too expensive, but if 20 people participate an everyone donates 10-12 € it works.

I will join in this, and here are the first 12 €.

Who keeps counting?

**Comment by Kohle81** [comment]
Schau mal,

hilft dir diese Information weiter?

Die ID vom Umweltsensor findet man wie bei alle anderen Sensoren mit:
http://ip-vom-homepilot//deviceajax.do?sensors=1

Zu finden ist er unter "description":"Umweltsensor"

Dann lassen sich alle Werte vom Umweltsensor mit
http://ip-vom-homepilot/device…meter=id-vom-umweltsensor
auslesen.

Code
{ "response" : "get_meter_data", "status" : "ok", "data":[{"Lichtwert":"xxxxx lx"},{"Windgeschw.":"x m/s"},{"Temperatur":"xx.x °C"},{"Regen":"Nicht erkannt"},{"Sonnenhöhe":"xx°"},{"Sonnenrichtung":"x-x (xx°)"},{"Aktualisiert":"xx.xx.xx - xx:xx"}] }

**Comment by bombadiltom** [comment]
Mal sehen: https://twitter.com/bombadil_tom/status/1053996084962516994

**Comment by Kohle81** [comment]
Wäre natürlich genial, wenn dies funktionieren würde. Ich nehm denen dann bestimmt einen ab......

Kurz noch eine Frage zu diesem Rademacher-Zubehör:

Rademacher 3500 08 64 Aktor https://www.amazon.de/dp/B007NIYG9S/ref=cm_sw_r_cp_api_u-hZBbRMJANJV

Das dürfte ja bereits jetzt schon über das plugin an homekit übergeben werden, oder?

Gruß Manuel

**Comment by bombadiltom** [comment]
Ja, davon habe ich zwei an den Jalousien im Wohnzimmer. Die funktionieren.

**Comment by Kohle81** [comment]
Kurze Frage zu den unterstützen Typen. In der Eve App werden z.b. auch andere Werte als die beiden genannten angezeigt. Homebridge könnte dies also.

Wären dann die anderen Werte in einer Drittanbieter-App zu verwenden?

Siehe z.b. bei diesem plugin:

https://github.com/naofireblade/homebridge-weather-plus/blob/master/README.md
